.ci_helpers/docker/setup-services.py

```diff
@@ -77,36 +89,38 @@ def run_commands(commands: List[Dict]) -> None:
 
 
 def load_s3(*args, **kwargs) -> None:
+    """Populate MinIO with test data from the Pooch cache (skip .zip files)."""
+    pooch_path = get_pooch_data_path()
     common_storage_options = dict(
         client_kwargs=dict(endpoint_url="http://localhost:9000/"),
         key="minioadmin",
         secret="minioadmin",
     )
     bucket_name = "ooi-raw-data"
```

Comment:
I think the original was because of some copy-paste in the initial file. Seems fitting to change it. :)
```suggestion
    bucket_name = "echo-test-data"
```